dbOperations/hooks/useUserInfo.js

```diff
@@ -31,7 +32,14 @@ export const useUserInfo = ({
           doc(db, "teams", currentTeamId, "users", userId),
         );
         const data = querySnapshot.data();
-        if (!data) {
+
+        // DEBUG REMOVE BELOW LOG LINES LATER
+        console.log("DATA");
+        console.log(data);
+        console.log("VERIFIED STATUS");
+        console.log(auth.currentUser.emailVerified);
+
+        if (!data || !auth.currentUser.emailVerified) {
```

Comment:
Shouldn't this use currentUserVerified instead? I think it's a nicer way of accessing this information

Reply:
Should be resolved by https://github.com/Golf-Drill-Challenge-App/Golf-App/pull/288/commits/69b6322cfc0d8564b927f921b64c41ba3a58fe00